docs/MO_DG/prepare_model/convert_model/supported_model_formats.md

```diff
@@ -16,32 +16,533 @@
    openvino_docs_MO_DG_prepare_model_convert_model_Convert_Model_From_Kaldi
    openvino_docs_MO_DG_prepare_model_convert_model_tutorials
 
-.. meta::
-   :description: In OpenVINO, ONNX, PaddlePaddle, TensorFlow and TensorFlow Lite 
-                 models do not require any prior conversion, while MxNet, Caffe and Kaldi do.
 
+**OpenVINO IR (Intermediate Representation)** - the proprietary and default format of OpenVINO, benefiting from the full extent of its features. All other model formats presented below will ultimately be converted to :doc:`OpenVINO IR <openvino_ir>`.
 
-**OpenVINO IR (Intermediate Representation)** - the proprietary format of OpenVINO™, benefiting from the full extent of its features.
+**ONNX, Pytorch, PaddlePaddle, TensorFlow, and TensorFlow Lite** may be used without any prior conversion and can be read directly by OpenVINO Runtime API by the use of ``read_model()`` or ``compile_model()``. Additional adjustment for the model can be performed using the ``convert_model()`` method, which allows you to set shapes, types or the layout of model inputs, cut parts of the model, freeze inputs etc. The detailed information of capabilities of  ``convert_model()`` can be found in :doc:`this <openvino_docs_MO_DG_Deep_Learning_Model_Optimizer_DevGuide>` article.
```

Comment:
Let change the order to: PyTorch, TensorFlow, ONNX, and PaddlePaddle.
TensorFlow Lite can be omitted since we can implicitly have TensorFlow covering Lite as well.

Reply:
I have changed it in the https://github.com/openvinotoolkit/openvino/pull/18495/commits/9afe744a3d8ed1ceff5797adac1f3f85ab8ce1e5 commit.